package.json

```diff
@@ -36,10 +36,9 @@
   },
   "dependencies": {
     "accessible-autocomplete": "^2.0.2",
-    "govuk-frontend": "^2.7.0",
-    "hof": "^19.14.13",
+    "hof": "^20.0.0-beta.39",
```

Comment:
can you take the master version please :)

Reply:
Updated to 20.1.17